test/unit/ObjectMakrTests/getFullProperties.js

```diff
@@ -0,0 +1,58 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("returns the full properties", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            var fullProperties = {
```

Comment:
Manually declaring full properties here defeats the purpose of creating a mock in the first place.

A better test would be to manually retrieve the full properties and test to ensure getFullProperties does return them.

However, this type of test is trivial and IMO unnecessary. Let's go with not testing any function that starts with `get` and is only a getter.